Assembler.UnitTests/FrameHandlers/EndFrameHandlerTests.cs

```diff
@@ -0,0 +1,144 @@
+﻿using System;
+using Assembler.Base.FrameHandlers;
+using Assembler.Core;
+using Assembler.Core.Entities;
+using Assembler.Core.Enums;
+using Moq;
+using NUnit.Framework;
+
+namespace Assembler.UnitTests.FrameHandlers
+{
+    [TestFixture]
+    public class EndFrameHandlerTests
```

Comment:
1. Change name of class (and file) to Final*
2. See comment of InitialFrameHandlerTests and apply here

Reply:
f

Reply:
f